### PR TITLE
remove `RCLogUtils`

### PR DESCRIPTION
**Pull request by aboedo**
removes `RCLogUtils`, along with its macros, and replaces them with the expanded calls.

### CODE REVIEW

File: Purchases/Public/RCPurchases.m [hunk 1]
```diff
@@ -586,7 +586,7 @@ - (void)        purchaseProduct:(SKProduct *)product
     } else if (payment.productIdentifier) {
         productIdentifier = payment.productIdentifier;
     } else {
-        RCLog(@"makePurchase - Could not purchase SKProduct. Couldn't find its product identifier. This is possibly an App Store quirk.");
+        [RCLog info:[NSString stringWithFormat:@"%@", RCStrings.purchase.could_not_purchase_product_id_not_found]];
```
Comment (aboedo) [inline Purchases/Public/RCPurchases.m:589]: this is the only change aside from expanding macros - we missed this string when moving stuff to `RCStrings` originally, so I added it now.

File: PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift [hunk 2]
```diff
@@ -33,4 +33,6 @@ import Foundation
         "SKPaymentTransaction missing an SKPayment - this is an issue with the App Store." }
     @objc public var skpayment_missing_product_identifier: String { "There is a problem with the SKPayment missing " +
         "a product identifier - this is an issue with the App Store." }
+    @objc public var could_not_purchase_product_id_not_found: String { "makePurchase - Could not purchase SKProduct. " +
```
Comment (taquitos) [inline PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift:36]: I don't see this used in our repo, or my search is terrible.

**Review by taquitos — APPROVED** [review]
Question about `could_not_purchase_product_id_not_found` but 👍.

File: PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift (hunk 2 above)
Comment (taquitos) [reply PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift:36]: Ah yes, my search is terrible.

**Pull request merged**